OrbitCore/Capture.cpp

```diff
@@ -200,6 +200,13 @@ bool Capture::StartCapture()
         GEventTracer.Start(GTargetProcess->GetID());
 #endif
     }
+    else if ( Capture::IsRemote() )
+    {
+        LinuxPerf perf(0);
+        Capture::NewSamplingProfiler();
+        Capture::GSamplingProfiler->StartCapture();
+        Capture::GSamplingProfiler->SetIsLinuxPerf(true);
```

Comment:
Shouldn't we set the LinuxPerf mode before starting the capture?

Reply:
I also addressed this. Somehow the Github UI does not update the change.